src/events/mod.rs

```diff
@@ -315,10 +315,13 @@ impl<'a> BytesStart<'a> {
     }
 
     /// Try to get an attribute
-    pub fn try_get_attribute(&'a self, attr_name: &[u8]) -> Result<Option<Attribute<'a>>> {
+    pub fn try_get_attribute<N: AsRef<[u8]> + Sized>(
```

Comment:
Attribute names don't need to be escaped right?  This would let users pass `&str`, `&[u8]`, or `Vec<u8>`.

Reply:
right, names don't need escaping.

Reply:
I noticed one difference between a library I'm writing and an existing one that uses libxml2 - the one based on libxml2 seems to strip tab characters `\t` out of attribute values.  I can't find anything in the spec that mentions that nor can I find it in libxml2 though - do you know whether that might be required?